src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(num) {
```

Comment:
The initial argument `num` is not necessary for the `makeInfinityAdder` function as per the task instructions. The sum should always start at 0, and the adder function should not take an initial value.